# Description the point on a path

geometry_msgs/Pose pose
float64 kappa
float64 dkappa
float64 s
float64 width   # 路点的宽度（指车道宽度）
Bound   bound   # 路点的边界（指整个路的边界，相对于该点的横向距离）